src/TransactionBuilder.cs

```diff
@@ -654,7 +654,8 @@ private LineItemModel GetMostRecentLine(string memberName = "")
         /// <returns></returns>
         public async Task<TransactionModel> CreateAsync()
         {
-            return await _client.CreateTransactionAsync(null, _model);
+            return await _client.CreateTransactionAsync(null, _model)
+                .ConfigureAwait(false);
```

Comment:
I've addressed the `ConfigureAwait()` issue throughout the codebase.  This will get patched in the June release.